build.sh

```diff
@@ -35,7 +35,7 @@ export DORIS_HOME=${ROOT}
 
 . ${DORIS_HOME}/env.sh
 
-PARALLEL=8
+PARALLEL=$(nproc)
```

Comment:
May be conservative when setting compile concurrency degree. Is PARALLEL = nproc/4 + 1 all right?